### DISCUSSION TITLE
CoralNet -> TagLab

### DISCUSSION THREAD

**Discussion by jebyrnes** [cnr-isti-vclab/TagLab#119] [Q&A]
Does anyone have a good workflow/tutorial for exporting from CoralNet and importing into TagLab? Is that possible? Would love to shift platforms for a dataset and leverage the newer AI tools!

**Comment by Matt-Davies-314** [comment]
I've not used any of these tools myself, but it looks like they might be just what you're looking for: https://github.com/Jordan-Pierce/CoralNet-Toolbox

**Comment by Jordan-Pierce** [comment]
@jebyrnes this will be a feature in the very near future! Check back soon.
